SafeguardSessions/TROUBLESHOOTING.md

```diff
@@ -6,6 +6,7 @@ When you import or refresh data with the Power BI Connector, you will likely fac
 
 - The data import finishes without any immediate signs of error.
 - A pop-up appears indicating an error.
+- An error occurs when connecting via the Power BI Service.

```

Comment:
Connecting <something> via the Power BI Service? 
Connecting <to something> (...)? 

My guess would be (...) connecting SPS via the (...), but not sure. :-)

Reply:
I'll look for an updated version of this in the next PR. :-)